.gitignore

```diff
@@ -1 +1 @@
-node_modules
+node_modules
```

Comment:
Better to have the newline at the end of file. Your editor probably did this after you removed .DS_Store.